Common/src/grid_movement_structure.cpp

```diff
@@ -7755,7 +7815,16 @@ void CSurfaceMovement::WriteFFDInfo(CGeometry *geometry, CConfig *config) {
       output_file << "FFD_DEGREE_I= " << FFDBox[iFFDBox]->GetlOrder()-1 << endl;
       output_file << "FFD_DEGREE_J= " << FFDBox[iFFDBox]->GetmOrder()-1 << endl;
       if (nDim == 3) output_file << "FFD_DEGREE_K= " << FFDBox[iFFDBox]->GetnOrder()-1 << endl;
-      
+      if (config->GetFFD_Blending() == BSPLINE_UNIFORM) {
```

Comment:
Aok. As long as we have a very clear error message (looks like you did this already), then it's ok.